.github/workflows/scanner-mapping-update.yaml

```diff
@@ -22,13 +22,7 @@ jobs:
       uses: google-github-actions/setup-gcloud@v2
 
     - name: Download repository mappings
-      run: |
-        curl --fail --silent --show-error --max-time 60 --retry 3 --create-dirs \
```

Comment:
The removal of `--create-dirs` is slightly concerning me. Need to test to make sure the bundling works as intented

Reply:
from what I can tell, it wasn't needed? Seems to work when I run this from within Scanner's CI image